### PR TITLE
doc: fix broken link

### PR DESCRIPTION
**Pull request by yaojingguo**
Release note: None

### CODE REVIEW

File: docs/first-pr.md [hunk 1]
```diff
@@ -632,7 +632,7 @@ Here's a checklist of action items to keep you sane:
 [production-ready code]: #production-ready-code
 [Reviewable]: https://reviewable.io
 [RocksDB]: http://rocksdb.org
-[STYLE.md]: /STYLE.md
+[style.md]: /docs/style.md
```
Comment (a-robinson) [inline docs/first-pr.md]: All the links to `style.md` in this file should be done in the same way (relative vs absolute). Mind changing this to just `style.md` match the others?

**Review by a-robinson — COMMENTED** [review]
Thanks @yaojingguo!

**Comment by yaojingguo** [comment]
@a-robinson I have created a new commit with the changes suggested by you. PTAL.

**Review by a-robinson — APPROVED** [review]
LGTM

**Pull request merged**